tests/components/rfxtrx/test_init.py

```diff
@@ -130,23 +130,17 @@ def record_event(event):
         calls.append(event)
 
     hass.bus.async_listen(rfxtrx.EVENT_BUTTON_PRESSED, record_event)
-    await hass.async_block_till_done()
-    entity = rfxtrx.RFX_DEVICES["213c7f2_16"]
-    entity.update_state(False, 0)
-    assert "Test" == entity.name
-    assert "off" == entity.state
-    assert entity.should_fire_event
-
-    event = rfxtrx.get_rfx_object("0b1100cd0213c7f210010f51")
-    event.data = bytearray(
-        [0x0B, 0x11, 0x00, 0x10, 0x01, 0x18, 0xCD, 0xEA, 0x01, 0x01, 0x0F, 0x70]
-    )
-    await _signal_event(hass, event)
-    await hass.async_block_till_done()
 
-    assert event.values["Command"] == "On"
-    assert "on" == entity.state
-    assert hass.states.get("switch.test").state == "on"
+    state = hass.states.get("switch.test")
+    assert state
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)